crowd_anki/importer/import_dialog.py

```diff
@@ -0,0 +1,189 @@
+            heading_ui.setFont(heading_font)
+            self.form.list_personal_fields.addItem(heading_ui)
+
+        def add_field(name, is_personal):
+            field_ui = QListWidgetItem(name)
+            field_ui.setCheckState(Qt.Checked if is_personal else Qt.Unchecked)
+            field_ui.setFlags(Qt.ItemIsEnabled | Qt.ItemIsUserCheckable)
+            self.form.list_personal_fields.addItem(field_ui)
+
+        for model in self.deck_json["note_models"]:
+            model_name = model["name"]
+            model_id = model[UUID_FIELD_NAME]
+            add_header(model_name)
+
+            for field in model["flds"]:
+                field_name = field["name"]
+                add_field(field_name, self.import_defaults.has_pf(field_name, model_name, model_id))
+
+    def setup_misc(self):
+        self.form.import_message_textbox.setText(self.import_defaults.import_message)
+
+        if self.import_defaults.suggest_tag_imported_cards:
+            self.form.cb_tag_cards.setCheckState(Qt.Checked)
+            self.form.cb_tag_cards.setText("Tag Cards (Suggested by Deck Manager!)")
+        # else:
+        # set as default from config settings
+
+        if self.userConfig.import_notes_ignore_deck_movement:
+            self.form.cb_ignore_move_cards.setCheckState(Qt.Checked)
+
+        def set_checked_and_text(checkbox, text, count, checked: bool = True):
+            checkbox.setCheckState(Qt.Checked if checked else Qt.Unchecked)
+            if count is not None:
+                text = f"{text}: {'{:,}'.format(count)}"
+            checkbox.setText(text)
+
+        set_checked_and_text(self.form.cb_headers, "Headers", None)
+        set_checked_and_text(self.form.cb_note_models, "Note Models", len(self.deck_json['note_models']))
+        set_checked_and_text(self.form.cb_notes, f"Notes", len(self.deck_json['notes']))
+        set_checked_and_text(self.form.cb_media, f"Media Files", len(self.deck_json['media_files']))
+
+        # TODO: Deck Parts to Use, check which are actually in the deck_json
+
+    def read_import_config(self):
+        config = ImportConfig(
+            add_tag_to_cards=string_cs_to_list(
+                self.form.textedit_tags.text()) if self.form.cb_tag_cards.isChecked() else [],
+
+            use_header=self.form.cb_headers.isChecked(),
+            use_note_models=self.form.cb_note_models.isChecked(),
+            use_notes=self.form.cb_notes.isChecked(),
+            use_media=self.form.cb_media.isChecked(),
+
+            ignore_deck_movement=self.form.cb_ignore_move_cards.isChecked()
+        )
+
+        self.read_personal_fields(config)
+
+        self.final_import_config = config
+
+    def read_personal_fields(self, config):
+        current_note_model = ""
+        current_uuid = ""
+        ui_fields: List[QListWidgetItem] = self.get_ui_pf_items()
+        for ui_field in ui_fields:
+            if not ui_field.flags() & Qt.ItemIsUserCheckable:  # Note Model Header
```

Comment:
could you save those 2 fields into some variable instead of cycling over all fields like this to find them?